### ISSUE TITLE
Alt-Tab and Google Chrome interaction

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jimradford/superputty#295]
```
What steps will reproduce the problem?
1. Open SuperPuTTY with several sessions (tabs)
2. Open Google Chrome with several tabs
3. ALT-TAB to give focus to Google Chrome
4. Select another tab
5. ALT-TAB to give focus to SuperPuTTY
6. SuperPuTTY has focus but is not brought to foreground

What is the expected output? What do you see instead?
SuperPuTTY should be brought to foreground with ALT-TAB toggle.

What version of the product are you using? On what operating system?
1.3.0.11

Please provide any additional information below.
I am not sure if other programs cause an issue. If I ALT-TAB to Chrome and then 
ALT-TAB back, no issue. Only when I select a different Chrome tab do I see the 
issue. Of course I am often toggling between these two Apps so I see it quite 
frequently. I set up F4 and F5 to toggle between tabs in SuperPuTTY and when I 
ALT-TAB to SuperPuTTY and it does not come to foreground, pressing F4 makes it 
come to foreground. That is why I think it has focus, despite not being visible.
```

Original issue reported on code.google.com by `tcraw...@gmail.com` on 14 Feb 2013 at 6:44

**Comment by shallot** [comment]
I have also been reproducing the same issue on Windows 8.1, and it's not related to Chrome, I don't use it.

The upgrade to Windows 10 changed the behavior somewhat - it's now apparent that SuperPutty is doing something strange: when you press Alt+Tab within SuperPutty, you see in the popup window that the SuperPutty window is still the one selected, and you have to press Tab another time to get it to move to the next window. But with any other Windows program, this happens by default, you press Alt+Tab only once and it instantly moves to the next window.

**Comment by jimradford** [comment]
This does appear to be a duplicate of #216
